crates/gateway-types/src/reply.rs

```diff
@@ -187,17 +187,18 @@ pub mod call {
 
 /// Used to deserialize replies to Starknet transaction requests.
 ///
-/// We only care about the statuses so we ignore other fields.
 /// Please note that this does not have to be backwards compatible:
 /// since we only ever use it to deserialize replies from the Starknet
 /// feeder gateway.
 #[serde_as]
```

Comment:
This annotation seems to be unnecessary..

```suggestion
#[serde_as]
```